lib/utils.js

```diff
@@ -141,6 +141,20 @@ function normalizeQueryConfig (config, values, callback) {
   return config
 }
 
+function handleQueryError (err, query) {
+  err.sql = {
+    text: query.text,
+    values: query.values
```

Comment:
Including values introduces a security issue. This could lead to user data e.g. passwords being logged in production. Perhaps this should be configurable or removed altogether.